app/index.html

```diff
@@ -52,7 +52,7 @@
     <![endif]-->
       <header>
         <section class="nav">
-            <a href="http://hood.ie" class="logo"><img src="images/logo.svg" width="135" alt=""></a>
+            <a href="http://hood.ie" class="logo"><img src="images/logo.svg" width="135" alt=""><b>Hoodie</b></a>
```

Comment:
Using `alt` does indeed work. However, the alt wouldn't show up as a text link if you were to disable CSS styles. I believe CSS as well as JS is an enhancement and if someone can't see the Hoodie Image they have no idea that there is a link there in the first place, unless they are solely using a keyboard :) So adding the actual text to the link (and hiding it offscreen) adds to the accessibility of the app, as well as providing all the possible visual content if CSS isn't loading.